public/stylesheets/site/2.0/15-group-comments.css

```diff
@@ -82,6 +82,15 @@ fieldset.comments, .comment .userstuff {
   margin: 1.286em 0.643em 0.25em;
 }
 
+.comment span.role {
+  font-weight: bold;
+  color: #900;
+}
+
+.comment.official {
+  background-color: #FCC;
+}
```

Comment:
We use lowercase letters in hex color codes and try to avoid chaining classes. However, this should be removed, since it wasn't requested in the issue.

Reply:
Sorry that was me testing the .official class, same with the colour in .role